src/components/ProjectListing.js

```diff
@@ -41,84 +41,84 @@ export const ProjectListing = ({
+                >
+                  <div>
+                    <IconButton aria-label="Copy link to clipboard" disabled={!project.public} onClick={() => {
+                      copy(`${window.location.host}/?id=${project.id}`)
+                      setCopied()
+                    }}>
+                      <LinkIcon />
+                    </IconButton>
+                  </div>
+                </Tooltip>
+                <Tooltip title="Load">
+                  <IconButton aria-label="Load" onClick={() => {
+                    load(firebase, project, setProject)
+                    handleHide()
                   }}>
-                    <LinkIcon />
+                    <LaunchIcon />
                   </IconButton>
-                </div>
-              </Tooltip>
-              <Tooltip title="Load">
-                <IconButton aria-label="Load" onClick={() => {
-                  load(firebase, project, setProject)
-                  handleHide()
-                }}>
-                  <LaunchIcon />
-                </IconButton>
-              </Tooltip>
-              <Tooltip title="Clone">
-                <IconButton aria-label="Clone" onClick={() => {
-                  load(firebase, project, setProject)
-                  save(firebase, firestore, {
-                    uid,
-                    docid: undefined, // force a new document to be created
-                    metadata: project.versions[0].metadata,
-                    proof: serialization
-                  }, setProjectID)
-                }}>
-                  <FilterNoneIcon />
-                </IconButton>
-              </Tooltip>
-              <Tooltip title="Delete">
-                <IconButton aria-label="Delete" onClick={() => {
-                  // delete db entry in firestore
-                  firestore.delete({ collection: 'projects', doc: project.id })
-                  // delete blob in firebase storage
-                  firebase.deleteFile(`${uid}/${project.id}/0.proof`)
-                }}>
-                  <DeleteIcon />
-                </IconButton>
-              </Tooltip>
-            </CardActions>
-          </Card>
-        </Project>
+                </Tooltip>
+                <Tooltip title="Clone">
+                  <IconButton aria-label="Clone" onClick={() => {
+                    load(firebase, project, setProject)
+                    save(firebase, firestore, {
+                      uid,
+                      docid: undefined, // force a new document to be created
+                      metadata: project.versions[0].metadata,
+                      proof: serialization
+                    }, setProjectID)
+                  }}>
+                    <FilterNoneIcon />
+                  </IconButton>
+                </Tooltip>
+                <Tooltip title="Delete">
+                  <IconButton aria-label="Delete" onClick={() => {
+                    showModal('deleteConfirmation', { project: project });
```

Comment:
It appears I accidentally changed all the indentation slightly. This is the only really important change (the other changes are boiler plate to get this to work

Reply:
Nevermind, I have fixed the indentation